code/game/objects/items/weapons/tools/saws.dm

```diff
@@ -54,24 +57,43 @@
 	icon_state = "chainsaw"
 	hitsound = WORKSOUND_CHAINSAW
 	worksound = WORKSOUND_CHAINSAW
-	force = WEAPON_FORCE_BRUTAL //Rip and tear!
+	force = WEAPON_FORCE_PAINFUL
+	switched_on_force = WEAPON_FORCE_BRUTAL //Rip and tear!
 	armor_penetration = ARMOR_PEN_SHALLOW
 	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTEEL = 10, MATERIAL_PLASTIC = 2)
-	tool_qualities = list(QUALITY_SAWING = 60, QUALITY_CUTTING = 50, QUALITY_WIRE_CUTTING = 20) //not the best choice to cut wires
+	switched_on_qualities = list(QUALITY_SAWING = 60, QUALITY_CUTTING = 50, QUALITY_WIRE_CUTTING = 20)	//not the best choice to cut wires
+	switched_off_qualities = list(QUALITY_SAWING = 15, QUALITY_CUTTING = 15, QUALITY_WIRE_CUTTING = 10)
+	toggleable = TRUE
 	max_upgrades = 4
 	use_fuel_cost = 0.1
 	max_fuel = 80
 
+/obj/item/weapon/tool/saw/chain/turn_on(mob/user)
```

Comment:
I might think they can be moved to /obj/item/weapon/tool/proc/turn_on(var/mob/user) since there's either fuel or cell only tools.